presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -36,40 +36,37 @@
 
 public class RuleAssert
 {
-    private final Metadata metadata;
-    private Session session;
     private final Rule rule;
 
     private final PlanNodeIdAllocator idAllocator = new PlanNodeIdAllocator();
+    private final LocalQueryRunner queryRunner;
```

Comment:
If it becomes a problem, we can always encapsulate them all in a context object of some sort.